styles.css

```diff
@@ -128,23 +175,1194 @@ ul {
+  padding: 12px;
+  width: 67px;
+  height: 40px;
+  background: #fff;
+  border-radius: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.txt42 {
+  width: 43px;
+  height: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #36b37e;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.tag43 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 79px;
+  height: 40px;
+  background: #fff;
+  border-radius: 8px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.txt43 {
+  width: 55px;
+  height: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #36b37e;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.tag44 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 61px;
+  height: 40px;
+  background: #fff;
+  border-radius: 8px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.txt44 {
+  width: 53px;
+  height: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #36b37e;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.tag45 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 77px;
+  height: 40px;
+  background: #fff;
+  border-radius: 8px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.txt45 {
+  width: 53px;
+  height: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #36b37e;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.icons_S {
+  position: absolute;
+  width: 327px;
+  height: 128px;
+  left: 50px;
+
+  /* top: 16px; */
+}
+
+.icon_lang_S {
+  position: absolute;
+  width: 80px;
+  height: 80px;
+  left: 124px;
+  top: 24px;
+}
+
+.img_placeholder_C {
+  width: 327px;
+  height: 352px;
+  box-sizing: border-box;
+
+  /* position: absolute; */
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 64px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 375px;
+  height: 879px;
+  left: 0;
+  top: 5483px;
+}
+
+.txtF {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 363px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.primary_txt_F {
+  width: 327px;
+  height: 363px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly adjust the alignment of both "Work " and "About me" section just like Figma design , Minimum mobile screen size should be 375px.

Your Project at Minimum mobile screen size of 375px Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/205458867-4a23d3ea-9f7a-43fd-98a0-ecd8685558bb.PNG)

![Capture](https://user-images.githubusercontent.com/93598189/205458923-9ba02ca7-bea8-46e9-a93c-8a11fb91773c.PNG)

The Figma design look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/205458978-21cbd884-b477-4b2c-9105-7413c90819e7.PNG)

![Capture](https://user-images.githubusercontent.com/93598189/205459058-36364f57-9baf-4a91-8d39-b3e6b695828c.PNG)